lib/spack/docs/tutorial_modules.rst

```diff
@@ -103,56 +365,66 @@ Next you should install a few modules that will be used in the tutorial:
+     autoconf-2.69-gcc-5.4.0-bvabhji      libtool-2.4.6-gcc-5.4.0-o2pfwjf              ncurses-6.0-gcc-7.2.0-oh6pqty               pkg-config-0.29.2-gcc-5.4.0-ae2hwm7       readline-7.0-gcc-5.4.0-gizxpch
+     automake-1.15.1-gcc-5.4.0-kaiefe4    lmod-7.7-gcc-5.4.0-okcwjgw                   netlib-lapack-3.6.1-gcc-7.2.0-5sywztc       pkg-config-0.29.2-gcc-7.2.0-76z7ehw       readline-7.0-gcc-7.2.0-eqos6rz
+     bzip2-1.0.6-gcc-7.2.0-mwamumj        lua-5.3.4-gcc-5.4.0-ytxw2gq                  netlib-scalapack-2.0.2-gcc-7.2.0-5lb2j5p    py-appdirs-1.4.3-gcc-7.2.0-7ncu7zr        sqlite-3.20.0-gcc-7.2.0-hfmjilk
+     cmake-3.9.4-gcc-7.2.0-6bxdr6h        lua-luafilesystem-1_6_3-gcc-5.4.0-5dzzlt4    netlib-scalapack-2.0.2-gcc-7.2.0-ax6aza6    py-numpy-1.13.1-gcc-7.2.0-22n5oub         tcl-8.6.6-gcc-5.4.0-767ls4i
+     gcc-7.2.0-gcc-5.4.0-go3z4hb          lua-luaposix-33.4.0-gcc-5.4.0-w5jpnwm        netlib-scalapack-2.0.2-gcc-7.2.0-c4v5l7j    py-packaging-16.8-gcc-7.2.0-c37cjmq       zlib-1.2.11-gcc-5.4.0-swly52a
+     gdbm-1.13-gcc-5.4.0-vdhoris          m4-1.4.18-gcc-5.4.0-r5envx3                  netlib-scalapack-2.0.2-gcc-7.2.0-m7rzcmh    py-pyparsing-2.2.0-gcc-7.2.0-ahdh5cx      zlib-1.2.11-gcc-7.2.0-lv5fabl
+     git-2.9.4-gcc-5.4.0-atwjs4i          mpc-1.0.3-gcc-5.4.0-tumbpsh                  openblas-0.2.20-gcc-7.2.0-kvddide           py-scipy-0.19.1-gcc-7.2.0-7hi7r5j
+     gmp-6.1.2-gcc-5.4.0-qc4qcfz          mpfr-3.1.5-gcc-5.4.0-mdi6irz                 openmpi-1.10.2-gcc-7.2.0-ufw7pdi            py-setuptools-35.0.2-gcc-7.2.0-cvasi7i
+     isl-0.18-gcc-5.4.0-vttqout           mpich-3.2-gcc-7.2.0-7gxffhv                  openssl-1.0.2k-gcc-7.2.0-pxv3dh4            py-six-1.10.0-gcc-7.2.0-3xk5mod
+     libsigsegv-2.11-gcc-5.4.0-fypapcp    ncurses-6.0-gcc-5.4.0-ukq4tcc                perl-5.24.1-gcc-5.4.0-mfzwy6y               python-2.7.14-gcc-7.2.0-555u7ea
 
-------------------------------------------------
-Filter unwanted modifications to the environment
-------------------------------------------------
+  Use "module spider" to find all possible modules.
+  Use "module keyword key1 key2 ..." to search for all possible modules matching any of the "keys".
 
-The non-hierarchical TCL module files that have been generated so far
-follow the default rules for module generation, which are given
-:ref:`here <modules-yaml>` in the reference part of the manual. Taking a
-look at the ``gcc`` module you'll see something like:
+The non-hierarchical module files that have been generated so far
+follow :ref:`the default rules for module generation <modules-yaml>`.
+Taking a look at the ``gcc`` module you'll see, for example:
 
 .. code-block:: console
 
-  $ module show gcc-6.2.0-gcc-4.8-twd5nqg
+  root@module-file-tutorial:/# module show gcc-7.2.0-gcc-5.4.0-go3z4hb
```

Comment:
The bash prompt will look different for every user, right? I think it's better to stay consistent with the rest of Spack's documentation and use `$` everywhere.

Reply:
Nope. It's a docker image. You'll see exactly that.